src/main/java/com/dumbdogdiner/stickyapi/common/util/BookUtil.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import com.dumbdogdiner.stickyapi.common.book.chat.JsonComponent;
+import lombok.NonNull;
+import org.commonmark.node.Document;
+import org.commonmark.node.Node;
+import org.commonmark.node.ThematicBreak;
+
+/**
+ * Utilities for text and books.
+ */
+public class BookUtil {
+    public static final int PIXELS_PER_LINE = 113;
+    public static final int LINES_PER_PAGE = 14;
+    public static final int PAGES_PER_BOOK = 50;
+
+    private static final Pattern SPLIT_PATTERN = Pattern.compile("(?<=[ \n])|(?=[ \n])");
+
+    // TODO: Test efficiency of this vs switch/case
```

Comment:
Putting this in a HashMap makes it easier to maintain and read, the efficiency doesn't matter too much here since there is almost no noticeable difference.